src/liquid/place/bulk_create_form.html

```diff
@@ -1,115 +1,91 @@
-<section class="section is-small">
-  <form 
-    id="form_place_create" 
-    hx-encoding='multipart/form-data' 
-    autocomplete="off"
-    hx-post="/place?op=bulk&type={{contactType.name}}"
-    hx-target="this" 
-    hx-swap="outerHTML"
-  >
-      <input name="place_type" value="{{contactType.name}}" hidden />
+<form class="data-form" id="form_place_create" hx-encoding='multipart/form-data' autocomplete="off"
+  hx-post="/place?op=bulk&type={{contactType.name}}" hx-target="this" hx-swap="outerHTML">
+  <input name="place_type" value="{{contactType.name}}" hidden />
 
-      {% if errors %}
-        <div class="notification is-danger">
-          {{errors.message}}
-        </div>
-      {% endif %}
-      
-      <div id="bulk-upload-file" class="file has-name is-centered">
-        <label class="file-label">
-          <input class="file-input" type="file" name="resume">
-          <span class="file-cta">
-            <span class="file-icon">
-              <i class="fas fa-upload"></i>
-            </span>
-            <span class="file-label">
-              Choose a file…
-            </span>
-          </span>
-          <span class="file-name">
-            No file uploaded
-          </span>
-        </label>
-      </div><br/>
-      <div class="field is-grouped is-grouped-centered">
-        <div class="control">
-            <button id="place_create_submit" class="button is-link">Add</button>
-        </div>
-        <div class="control">
-            <a href="/" id="cancel" class="button">Cancel</a>
-        </div>
-      </div>
+  {% if errors %}
+  <div class="error-bg">
+    {{errors.message}}
+  </div>
+  {% endif %}
 
-      <div class="tile box">
-        <table class="table is-fullwidth is-striped is-hoverable">
-          <thead>
-            <tr>
-              <th>Expected Column</th>
-              <th>Required</th>
-              <th>Description</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>{{ contactType.replacement_property.friendly_name }}</td>
-              <td>false</td>
-              <td>Name of a <i>{{ contactType.friendly }} to replace</i> (leave blank to create new)</td>
-            </tr>
+  <div  style="width: 70%; margin-bottom: 10px;" id="bulk-upload-file">
+    <input class="file-input" type="file" name="resume">
+  </div>
+  <div>
+    <button id="place_create_submit" class="btn">Add</button>
+    <a href="/" id="cancel" class="clear-btn">Cancel</a>
```

Comment:
These two buttons should be on the same visual level but use the of class `btn` adds a shadow to the `Add` button